docs/developing/dev-test.md

```diff
@@ -0,0 +1,3 @@
+* Test your build
```

Comment:
Add some general guidelines on getting your build deployed, some standard flows to go through, where to look for success/failure/diagnostic info